tests/data_context/conftest.py

```diff
@@ -596,7 +611,7 @@ def _mocked_get_response(*args, **kwargs):
             {
                 "data": {
                     "attributes": {
-                        "datasource_config": datasource_config_with_names_and_ids,
+                        "datasource_config": datasource_config_with_names_and_ids.to_json_dict(),
```

Comment:
This was a bug - the JSON response will not contain rich types

Reply:
Good catch!

Reply:
Yes, thank you!